fabric8-docker-registry/src/main/fabric8/fabric8-docker-registry-storage-pvc.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fabric8-docker-registry-storage
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
Well, personally I would use 100Gi for all PVs in Fabric8, but that depends on the needs of each user. Might be configurable through gofabric8's parameters, but that's a feature for the future I think :) I change it to 5Gi for now.